internal/volumes/mount.go

```diff
@@ -134,7 +139,20 @@ func (s *LinuxMountService) Publish(targetPath string, devicePath string, opts M
 		return s.mounter.Mount(devicePath, targetPath, opts.FSType, mountOptions)
 	}
 
-	return s.mounter.FormatAndMount(devicePath, targetPath, opts.FSType, mountOptions)
+	formatOptions := make([]string, 0)
+
+	if opts.FormatOptions != "" {
+		lexer := shell.NewLex('\\')
+		formatOptions, err = lexer.ProcessWords(opts.FormatOptions, shell.EnvsFromSlice([]string{}))
+		if err != nil {
+			return err
+		}
+	} else if opts.FSType == "xfs" {
+		formatOptions = append(formatOptions, "-c")
+		formatOptions = append(formatOptions, fmt.Sprintf("options=%s", XFSDefaultConfigPath))
```

Comment:
```suggestion
		formatOptions = append(formatOptions, "-c",  fmt.Sprintf("options=%s", XFSDefaultConfigPath))
```

Append has a variadic elements argument.